automation/repeated_test.sh

```diff
@@ -80,7 +80,7 @@ if (( $# > 0 )); then
         shift
     done
 else
-    TEST_LANES=( 'k8s-1.14' 'k8s-1.15' 'k8s-1.16' )
+    TEST_LANES=( "$( find cluster-up/cluster -name 'k8s-[0-9].[0-9][0-9]' | sort -r | grep -oE 'k8s.*' | head -3 )" )
```

Comment:
would it catch kind ones ? do we want to catch them ?

Reply:
no, we only want the k8s lanes. The output on my system is as follows:

```
> find cluster-up/cluster -name 'k8s-[0-9].[0-9][0-9]'
cluster-up/cluster/k8s-1.17
cluster-up/cluster/k8s-1.16
cluster-up/cluster/k8s-1.15
cluster-up/cluster/k8s-1.14
```

Reply:
ok thanks, and just 3 , well maybe we need 4 ?
because David Vossel said once that kubevirt should support 2 back, and sometimes we are already developing one forward, so its 4

Reply:
For the use case of finding flaky tests three should be good enough, I think. If required we coul djust adjust the parameters. Please bear in mind that in this lane the tests are run three times (by default, see [docs](https://github.com/kubevirt/kubevirt/blob/master/automation/repeated_test.sh#L7) for options) against each of the providers. The other lanes already test all the other k8s versions and flavors.

Reply:
sgtm
/lgtm
Thanks

Reply:
Nit: maybe `sort -rV`?

Reply:
Good point!